Compliance timeout updates (#56373)
openshift/openshift-docs: modules/compliance-timeout.adoc

timeout: '10m0s' <1>
maxRetryOnTimeout: 3 <2>
----

> <1> The `timeout` variable is defined as a duration string, such as `1h30m`. The default value is `30m`. The timeout can be disabled by setting the value to `0s`.

Reviewer: ```suggestion
<1> The `timeout` variable is defined as a duration string, such as `1h30m`. The default value is `30m`. To disable the timeout, set the value to `0s`.
```

Author: Yep, I agree. I like that a lot better. Thank you!
